site/content/en/blog/releases/1.6.0-rc.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "1.6.0 - Release Candidate"
+linkTitle: "1.6.0-rc"
+date: "2020-05-19"
+---
+This is the v1.6.0-rc release of Agones.
+
+This release includes **several breaking changes** to be aware of:
+
+- Upgrade to required Kubernetes version to 1.15
+- Alpha Multi Cluster allocator has been renamed, refactored slightly and moved to Stable.
```

Comment:
@pooneh-m just confirming language here on MC allocation -- this sound good to you?

Reply:
Sounds great. Thanks! 🙏

Reply:
Cool. Reworded things slightly here to reflect below, PTAL.